### PR TITLE
SublimeGit is open source, point links at github

### PR DESCRIPTION
**Pull request by buildersbrewery**
This PR fixes the startup screen within Sublime Text, when installing through Package Control. See commit for details.

### CODE REVIEW

File: messages/install.txt [hunk 1]
```diff
@@ -33,12 +33,12 @@ If you are on windows, or don't usually use git from the command
 line, there might be some things you'll need to set up. Go here
 for instructions on how to get set up:
 
-    https://docs.sublimegit.net/quickstart.html
+    https://github.com/SublimeGit/SublimeGit/blob/master/docs/quickstart.rst
```
Comment (miped) [inline messages/install.txt]: The documentation is available at readthedocs, which would be more appropriate: https://sublimegit.readthedocs.io/en/latest/quickstart.html

File: messages/install.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ Tutorial
 --------
 For the full tutorial, go to:
 
-    https://docs.sublimegit.net/tutorial.html
+    https://github.com/SublimeGit/SublimeGit/blob/master/docs/tutorial.rst
```
Comment (miped) [inline messages/install.txt]: Same as below: https://sublimegit.readthedocs.io/en/latest/tutorial.html

File: messages/install.txt [hunk 3]
```diff
@@ -33,12 +33,12 @@ If you are on windows, or don't usually use git from the command
...
+    https://github.com/SublimeGit/SublimeGit/blob/master/docs/quickstart.rst
 
 
 Issues, Questions or Bugs?
 --------------------------
 Send a mail to support@sublimegit.net, or report it in the issue
 tracker:
 
-    https://github.com/SublimeGit/SublimeGit/issues
+    https://github.com/SublimeGit/SublimeGit/issues/new
```
Comment (miped) [inline messages/install.txt]: I don't think sending people directly to the new issue is a good idea. People should try and look over existing issues before filing a new one.

**Comment by miped** [comment]
Made a few comments. With those edits, it's an easy merge.

**Comment by buildersbrewery** [comment]
As I deleted the forked repo on my end there're two possibilities now.

* I can redo the PR.
* You merge and change whatever you commented on.

**Pull request closed without merging**